i18n/en.json

```diff
@@ -5213,7 +5213,7 @@
   },
   {
     "id": "store.sql.schema_version.critical",
-    "translation": "The database schema version of %v cannot be upgraded.  You must not skip a version."
+    "translation": "The database schema version of %v is not supported. This Mattermost server supports schema versions 3.0.0 through %v."
```

Comment:
@crspeller does this message make sense? Just wondering if you know of a case where this message might be output for a version we do support (like when a server tries to go from 3.9 to 3.8, for example)